website/components/info.tsx

```diff
@@ -72,10 +72,10 @@ const SpecsContent: React.FC = () => (
       <p>
         Nest runs on a{" "}
         <Link
-          href="https://www.hetzner.com/dedicated-rootserver/ex44/"
+          href="https://www.hetzner.com/dedicated-rootserver/ex101/"
           className="text-HCPurpleText hover:underline"
         >
-          Hetzner EX44
+          Hetzner EX101
```

Comment:
Could you also update the specifications (memory, CPU, disk) and upgrade the PVE version (we're currently on 8.2.2)?